securesystemslib/signer.py

```diff
@@ -138,10 +132,10 @@ class SSlibSigner(Signer):
 
             The public and private keys are strings in PEM format.
     """
+
     def __init__(self, key_dict: Dict):
         self.key_dict = key_dict
```

Comment:
This `key_dict` can also split into multiple attributes like `keyid`, `keytype`, `scheme`, and `keyval` and have functions like `to_dict` and `to_securesystemslib_key` similar to `Key` implemented above.

Reply:
Got it 👍🏻.